### PR TITLE
Update: list picker component header name

### PR DESCRIPTION
**Pull request by tchcxp**
As discussed with @omgaz , we update the header name to `addon`.

### CODE REVIEW

**Comment by sharmajyo** [comment]
LGTM

File: src/components/Main.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ class AppComponent extends React.Component {
     const listPickerItemHeaders = {
       left: 'Team',
       right: 'Member',
-      optional: 'Required',
+      addon: 'Required',
```
Comment (mcslash) [inline src/components/Main.js]: Add on is two words or hyphenated, therefore it should probably be `addOn` (camel case).

Comment (omgaz) [inline src/components/Main.js]: We use addon internally as does bootstrap.

**Comment by omgaz** [comment]
LGTM merging 🍜  🍥

**Pull request merged**